test/data/mock.php

```diff
@@ -196,22 +215,24 @@ protected function testHTML( $req ) {
 	}
 
 	protected function cspFrame( $req ) {
-		// This is CSP only for browsers with "Content-Security-Policy" header support
-		// i.e. no old WebKit or old Firefox
 		header( "Content-Security-Policy: default-src 'self'; report-uri ./mock.php?action=cspLog" );
 		header( 'Content-type: text/html' );
 		echo file_get_contents( __DIR__ . '/csp.include.html' );
 	}
 
 	protected function cspNonce( $req ) {
-		// This is CSP only for browsers with "Content-Security-Policy" header support
-		// i.e. no old WebKit or old Firefox
 		$test = $req->query['test'] ? '-' . $req->query['test'] : '';
 		header( "Content-Security-Policy: script-src 'nonce-jquery+hardcoded+nonce'; report-uri ./mock.php?action=cspLog" );
 		header( 'Content-type: text/html' );
 		echo file_get_contents( __DIR__ . '/csp-nonce' . $test . '.html' );
 	}
 
+	protected function cspAjaxScript( $req ) {
+		header( "Content-Security-Policy: script-src 'self'; report-uri ./mock.php?action=cspLog" );
+		header( 'Content-type: text/html' );
+		echo file_get_contents( __DIR__ . '/csp-ajax-script.html' );
+	}
+
```

Comment:
This doesn't make sense here; the referenced file doesn't even exist on `3.x-stable`.

Reply:
Removed